public/gameInfo.php

```diff
@@ -1085,25 +1070,54 @@ function ResetProgress() {
-                if ($user !== null && $numAchievements == 0 && $flags == $officialFlag) {
-                    echo "<div>";
-                    echo "<div class='gameRequestsLabel'></div>";
-                    echo "<div><button type='button' class='btn setRequestLabel hidden'>Request Set</button></div>";
-                    echo "<div class='userRequestsLabel'></div>";
+                echo "<br>";
+                echo "<br>";
+
+                if (isset($user) && $numAchievements > 0) {
+                    echo "<div class='md:float-right'>";
+                    RenderGameProgress($numAchievements, $numEarnedCasual, $numEarnedHardcore);
                     echo "</div>";
                 }
 
+                if ($flags == $unofficialFlag) {
+                    echo "There are <b>$numAchievements Unofficial</b> achievements worth <b>$totalPossible</b> <span class='TrueRatio'>($totalPossibleTrueRatio)</span> points.<br>";
+                } else {
+                    echo "There are <b>$numAchievements</b> achievements worth <b>$totalPossible</b> <span class='TrueRatio'>($totalPossibleTrueRatio)</span> points.<br>";
+                }
+
                 if ($user !== null && $numAchievements > 0) {
-                    $renderRatingControl('Game Rating', 'ratinggame', 'ratinggamelabel', $gameRating[RatingType::Game]);
+                    if ($numEarnedHardcore > 0) {
+                        echo "You have earned <b>$numEarnedHardcore</b> HARDCORE achievements, worth <b>$totalEarnedHardcore</b> <span class='TrueRatio'>($totalEarnedTrueRatio)</span> points.<br>";
+                        if ($numEarnedCasual > 0) { // Some Hardcore earns
+                            echo "You have also earned <b> $numEarnedCasual </b> SOFTCORE achievements worth <b>$totalEarnedCasual</b> points.<br>";
+                        }
+                    } elseif ($numEarnedCasual > 0) {
+                        echo "You have earned <b> $numEarnedCasual </b> SOFTCORE achievements worth <b>$totalEarnedCasual</b> points.<br>";
+                    } else {
+                        echo "You have not earned any achievements for this game.<br/>";
+                    }
                 }
 
-                echo "</div>";
+                if ($user !== null && $numAchievements > 0) {
+                    if ($numEarnedCasual > 0 || $numEarnedHardcore > 0) {
+                        echo "<div class='devbox mb-4'>";
+                        echo "<span onclick=\"$('#resetboxcontent').toggle(); return false;\">Reset Progress ▼</span>";
+                        echo "<div id='resetboxcontent' style='display: none'>";
+                        echo "<button class='btn btn-danger' type='button' onclick='ResetProgress()'>Reset your progress for this game</button>";
+                        echo "</div></div>";
+                    }
+                }
+
+                echo "<br>";
```

Comment:
I'd use tailwind's margin classes instead of breaks. I was trying to reduce that `<br>` usage until they're gone.

Reply:
Done! Removed a few br tags and used divs with margins to achieve the same effect.